refellips/structureSE.py

```diff
@@ -346,6 +346,86 @@ def complex(self, wavelength):
         return real + 1j * 0.0
 
 
+class Sellmeier(ScattererSE):
+    r"""
+    Dispersion curves for Sellmeier oscillators.
+
+    Parameters
+    ----------
+    Am: {float, Parameter, sequence}
+        Amplitude of Sellmeir
+    En: {float, Parameter, sequence}
+        Center energy of oscillator
+    P: {float, Parameter, sequence}
+        Position of a pole (eV)
+    Einf: {float, Parameter}
+        Offset term
+    wavelength : float
+        default wavelength for calculation (nm)
+    name : str, optional
+        Name of material.
+
+    Notes
+    -----
+    Calculates dispersion curves for *k* oscillators, as implemented in CompleteEase.
+
+    ..math::
+
+    n = \sqrt{ \varepsilon (\infty) + \frac{A \lambda^2}{\lambda^2 - B^2} - E\lambda^2}
+
+    Examples
+    --------
+    >>> # Create a Lorentz oscillator
```

Comment:
Please adjustthe docstring for sellmeier, not Lorentz. Do you have a ref for sellmeier?

Reply:
I did update (mostly), just missed that one. All Sellmeier info came out of the CompleteEase manual